admin/src/pages/Creation.vue

```diff
@@ -18,6 +18,13 @@
           :creation="creation"
           @update-item="updateItem"
         />
+        <b-pagination
+          pills
+          v-model="currentPage"
+          per-page="25"
+          :total-rows="rows"
+          align="center"
+        ></b-pagination>
```

Comment:
This is only for me as question to understand:
Why do you create a b-pagination? Is it a bootstrap component that shows the page on which we are and the next pages?

Reply:
https://bootstrap-vue.org/docs/components/pagination